src/gossip/service.zig

```diff
@@ -759,18 +754,17 @@ pub const GossipService = struct {
     /// this includes sending push messages, pull requests, and triming old
     /// gossip data (in the gossip_table, active_set, and failed_pull_hashes).
     fn buildMessages(self: *Self) !void {
-        var last_push_ts: u64 = 0;
-        var last_stats_publish_ts: u64 = 0;
-        var last_pull_req_ts: u64 = 0;
+        var loop_timer = sig.time.Timer.start() catch unreachable;
+        var push_timer = sig.time.Timer.start() catch unreachable;
+        var pull_req_timer = sig.time.Timer.start() catch unreachable;
+        var stats_publish_timer = sig.time.Timer.start() catch unreachable;
```

Comment:
Is there a disadvantage to using `try` here? Returning an error seems preferable to `unreachable` when performance is not a concern.